examples/wordcloud_cn.py

```diff
@@ -63,11 +66,54 @@ def jieba_processing_txt(text):
     return ' '.join(mywordlist)
 
 
-wc = WordCloud(font_path=font_path, background_color="white", max_words=2000, mask=back_coloring,
-               max_font_size=100, random_state=42, width=1000, height=860, margin=2,)
+# The function for processing text with HaanLP
+def pyhanlp_processing_txt(text, isUseStopwordsOfHanLP=True):
+    CustomDictionary = JClass("com.hankcs.hanlp.dictionary.CustomDictionary")
```

Comment:
JClass is not defined?

Reply:
If use hanlp must install jpype1 .I forgot about it......

"pip install jpype1"......

Reply:
Pyhanlp provides a python interface for hanlp. Hanlp is currently the best-performing open source Chinese natural language processing class library, but it is implemented in Java, so we must use jpype to call Java classes.

Reply:
Ok but you must also import it, right?

Reply:
You need to make sure the continuous integration passes.

Reply:
I don't need to import it.The pyhanlp import jpype1 in the __init__.py.

```python
from jpype import JClass, startJVM, getDefaultJVMPath, isThreadAttachedToJVM, attachThreadToJVM
```

Reply:
If use pyhanlp,In line 114 there's a line like “from pyhanlp import  *“
If use jieba,In line 106 there's a line like “import jieba“。This increases efficiency, and the code is feasible.